src/bp/core/services/cms.ts

```diff
@@ -273,10 +273,31 @@ export class CMSService implements IDisposeOnExit {
 
     await this.broadcastRemoveElements(botId, ids)
 
+    this.deleteMedia(botId, elements)
+
     const contentTypes = _.uniq(_.map(elements, 'contentType'))
     await Promise.mapSeries(contentTypes, contentTypeId => this._writeElementsToFile(botId, contentTypeId))
   }
 
+  getMediaFiles(formData): string[] {
+    const iterator = (result: string[], value, key: string) => {
+      if (key.startsWith('image') && value.includes('/media/')) {
```

Comment:
can you extract '/media/' in a variable and use this variable .length instead of hardcoding + 7